src/main/java/org/zalando/problem/validation/ValidationProblemModule.java

```diff
@@ -0,0 +1,36 @@
+package org.zalando.problem.validation;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.core.util.VersionUtil;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.zalando.problem.spring.web.advice.validation.ConstraintViolationProblem;
+import org.zalando.problem.spring.web.advice.validation.Violation;
+
+/**
+ * A companion to {@link org.zalando.problem.ProblemModule} to enable serialization
+ * of {@link ConstraintViolationProblem} and {@link Violation} without relying on autodetection.
+ */
+public class ValidationProblemModule extends Module {
```

Comment:
I wasn't sure about the name myself. I derived it from the package name, where both `ConstraintViolationProblem` and `Violation` are located.
Will change...